library/src/main/java/com/flipkart/layoutengine/builder/DataParsingLayoutBuilder.java

```diff
@@ -233,27 +205,17 @@ private JsonElement findAndReplaceValues(JsonElement element, ParserContext pars
                         String formatterName = regexMatcher.group(2);
 
                         String formattedValue;
-                        try {
-                            formattedValue = format(Utils.getElementFromData(dataPath,
-                                    parserContext.getDataContext().getDataProvider(),
-                                    parserContext.getDataContext().getIndex()),
-                                    formatterName);
-                        } catch (JsonNullException | NoSuchDataPathException | InvalidDataPathException e) {
-                            if (ProteusConstants.isLoggingEnabled()) {
-                                logger.error(TAG_ERROR + "#findAndReplaceValues() " + e.getMessage());
-                            }
+                        result = Utils.getElementFromData(dataPath, parserContext.getDataContext().getDataProvider(), parserContext.getDataContext().getIndex());
+                        if (result.isSuccess() && null != result.element) {
```

Comment:
since `Result.element` is `@Nullable` it gives a warning. I could remove it, because `Result.isSuccess` kind of guarantees that `Result.element` will not be `null`.